crates/forge/tests/cli/test_cmd.rs

```diff
@@ -275,7 +275,10 @@ Ran 1 test suite [ELAPSED]: 1 tests passed, 0 failed, 0 skipped (1 total tests)
 });
 
 const SIMPLE_CONTRACT: &str = r#"
+pragma solidity 0.8.18;
```

Comment:
pls dont add pragma, its done in test project

Reply:
noted, thing here is that by importing console.sol test failed with
```
Found incompatible versions:
src/Simple.t.sol =0.8.27 imports:
    src/test.sol >=0.4.23
    src/console.sol =0.8.18

```
hence the quick fix

Reply:
I see, it should be relaxed in console.sol

Reply:
https://github.com/foundry-rs/foundry/pull/9078